docs/releases/2.0.txt

```diff
@@ -250,6 +254,14 @@ Database backend API
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
 
+* Increased the number of rows fetched by :meth:`.QuerySet.iterator` at each
```

Comment:
:meth:`.QuerySet.iterator` now fetches 2000 rows at a time instead of 100.

(the rationale should also be explained)